frontend/src/components/ToggleRow/ToggleRow.tsx

```diff
@@ -0,0 +1,39 @@
+import { Box, Stack, Text } from '@highlight-run/ui'
+
+import Switch from '@/components/Switch/Switch'
+
+export const ToggleRow = (
+	label: string,
+	info: string | undefined,
```

Comment:
Realize you're just moving this component but can `info` be required? i.e. can we remove `| undefined`?